build.gradle.kts

```diff
@@ -27,6 +27,7 @@ buildscript {
     classpath(Plugins.mapboxSdkVersionsPlugin)
     classpath(Plugins.pitestPlugin)
     classpath(Plugins.playPublisher)
+    classpath("com.github.ben-manes:gradle-versions-plugin:0.42.0")
```

Comment:
why not put in plugins as all entries above?

Reply:
done